src/main/java/core/basesyntax/WorkWithBytes.java

```diff
@@ -1,7 +1,19 @@
 package core.basesyntax;
 
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithBytes {
     public void writeBytesToFile(String fileName, byte[] data) {
-
+        File fileOfBytes = new File(fileName);
+        try (BufferedWriter bufferedWriter = new BufferedWriter(new FileWriter(fileOfBytes))) {
```

Comment:
```suggestion
        try (BufferedWriter bufferedWriter = new BufferedWriter(new FileWriter(fileName))) {
```
I guess that Sofia meant that this should work without creating File object